src/jobsautoreport/report.py

```diff
@@ -1,31 +1,151 @@
-from datetime import datetime, timedelta
-from typing import Optional
+import logging
+from datetime import datetime
+from typing import Any
 
+from jobsautoreport.models import JobState, JobType
 from jobsautoreport.query import Querier
 
+logger = logging.getLogger(__name__)
+
+
+class Job:
```

Comment:
I don't understand the role of this class, both of the methods could be in the Reporter class and take the job name as an extra parameter.